Core/src/com/serotonin/m2m2/db/H2Proxy.java

```diff
@@ -144,24 +197,46 @@ private void upgradePageStore(String propertyPrefix) {
                 String runScript = fullUrl + ";init=RUNSCRIPT FROM '" + dumpPath.toString() + "' COMPRESSION ZIP";
                 try(Connection conn = DriverManager.getConnection(runScript, user, password);){
                     Statement stat = conn.createStatement();
+                    
+                    //Update our version
+                    ResultSet rs = stat.executeQuery(H2_CREATE_VERSION_SELECT);
+                    if(rs.next()) {
+                        int version = rs.getInt(1);
+                        h2Properties.put("h2Version", Integer.toString(version));
+                    }
+                    
                     //Might as well do a compaction here
                     stat.execute("shutdown compact");
```

Comment:
Inconsistent style. literal SQL should be ALL CAPS right?

Reply:
YES IT SHOULD